### PR TITLE
88948 Issue when selecting the directions link for CC appointments

### PR DESCRIPTION
**Pull request by vbahinwillit**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

## Summary
This PR fixes issue with null destination address for CC appointments when clicking the 'Destination' link.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#88948

## Testing done
Unit testing

## Screenshots
Anchor tab should have populated 'daddr' attribute.

```
<span><a href="https://maps.google.com?saddr=Current+Location&amp;daddr=4810 PENN AVE, SINKING SPRINGS, PA 19608" target="_blank" rel="noopener noreferrer" aria-label="Directions to undefined Link opens in a new tab.">Directions</a></span>
```

## What areas of the site does it impact?
VAOS

## Acceptance criteria
- [ ] When clicking the 'Destination' link for a CC appointment, Google maps should open in a new tab with directions to the CC location.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/vaos/components/layout/CCLayout.jsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export default function CCLayout({ data: appointment }) {
               <Address address={address} />
               <div className="vads-u-margin-top--1 vads-u-color--link-default">
                 <va-icon icon="directions" size="3" srtext="Directions icon" />{' '}
-                <FacilityDirectionsLink location={address} />
+                <FacilityDirectionsLink location={{ address }} />
```
Comment (JunTaoLuo) [inline src/applications/vaos/components/layout/CCLayout.jsx:75]: Just curious if we need to make the same change on link 72?

**Review by JunTaoLuo — COMMENTED** [review]
One small question.

File: src/applications/vaos/components/layout/CCLayout.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/layout/CCLayout.jsx:75]: Good point. I just checked and it fine as is. The 'FacilityDirectionsLink' component expect a struct containing address and 'Address' doesn't.

Comment (JunTaoLuo) [reply src/applications/vaos/components/layout/CCLayout.jsx:75]: Ah interesting. I need to read up on what the differences between the different types of brackets mean.

**Review by JunTaoLuo — APPROVED** [review]

**Pull request merged**